test/test_choices.cpp

```diff
@@ -155,3 +155,43 @@ TEST_CASE("Parse multiple arguments that are not in fixed number of allowed "
       "Invalid argument \"6\" - allowed options: {1, 2, 3, 4, 5}",
       std::runtime_error);
 }
+
+TEST_CASE("Parse multiple arguments that are in range of allowed "
+          "INTEGER choices (Min Range case)" *
+          test_suite("choices")) {
+  argparse::ArgumentParser program("test");
+  program.add_argument("indices").nargs(1, 3).choices(1, 2, 3, 4, 5);
+
+  program.parse_args({"test", "1"});
```

Comment:
What is this unit test testing? Can you add some asserts to make sure that the parsed values are correct?

Reply:
My mistake, I just copied paste from existing test cases blindly.

Reply:
done.